src/turnkeyml/cli/report.py

```diff
@@ -78,11 +78,34 @@ def summary_spreadsheets(args) -> None:
 
                         # Copy the build-specific stats
                         for key, value in build.items():
-                            # Break each value in "completed build stages" into its own column
-                            # to make analysis easier
+                            # Break each value in "completed build stages" into state and duration
```

Comment:
```suggestion
                            # Break each value in "completed build stages" into status and duration
```

We're using "status" to describe the outcome of a function everywhere else (there is even a `filesystem.FunctionStatus(Enum)`). Any specific reason you're using "state" here? We already have a `State` class that means something very different.

Reply:
Changed to state status